integration/spark/shared/src/main/java/io/openlineage/spark/agent/util/PlanUtils.java

```diff
@@ -219,20 +219,34 @@ public static OpenLineage.DatasourceDatasetFacet datasourceFacet(
    * and namespace.
    *
    * @param parentRunId
-   * @param parentJob
+   * @param parentJobName
    * @param parentJobNamespace
    * @return
    */
   public static OpenLineage.ParentRunFacet parentRunFacet(
-      UUID parentRunId, String parentJob, String parentJobNamespace) {
+      UUID parentRunId,
```

Comment:
Limiting amount of args from 6 to 2 would be great.

Reply:
Same comment as above